### ISSUE TITLE
pub build should also produce dart2dart compilation

### ISSUE THREAD

**Issue by DartBot** [dart-lang/pub#705]
<a href="https://github.com/butlermatt"><img src="https://avatars.githubusercontent.com/u/1148886?v=3" align="left" width="96" height="96"hspace="10"></img></a> **Issue by [butlermatt](https://github.com/butlermatt)**
_Originally opened as dart-lang/sdk#14788_

----

Per the discussion on this thread:
https://groups.google.com/a/dartlang.org/d/topic/misc/8O1ABCQqTZY/discussion

It would be nice to have pub build also run dart2dart with minification and place that in the build directory as well. Particularly as we approach 1.0.

**Comment by DartBot** [comment]
<a href="https://github.com/nex3"><img src="https://avatars.githubusercontent.com/u/188?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [nex3](https://github.com/nex3)**

----

*Added this to the [Later](../milestones/Later) milestone.*
*Added [Area-Pub](../labels/Area-Pub), [Triaged](../labels/Triaged) labels.*

**Comment by DartBot** [comment]
<a href="https://github.com/munificent"><img src="https://avatars.githubusercontent.com/u/46275?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [munificent](https://github.com/munificent)**

----

*Removed [Type-Defect](../labels/Type-Defect) label.*
*Added [Type-Enhancement](../labels/Type-Enhancement) label.*

**Comment by DartBot** [comment]
<a href="https://github.com/munificent"><img src="https://avatars.githubusercontent.com/u/46275?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [munificent](https://github.com/munificent)**

----

*Removed [Priority-Unassigned](../labels/Priority-Unassigned) label.*
*Added [Priority-Medium](../labels/Priority-Medium) label.*

**Comment by DartBot** [comment]
<a href="https://github.com/kasperl"><img src="https://avatars.githubusercontent.com/u/2156198?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [kasperl](https://github.com/kasperl)**

----

*Removed this from the [Later](../milestones/Later) milestone.*
*Added [Oldschool-Milestone-Later](../labels/Oldschool-Milestone-Later) label.*

**Comment by DartBot** [comment]
<a href="https://github.com/kasperl"><img src="https://avatars.githubusercontent.com/u/2156198?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [kasperl](https://github.com/kasperl)**

----

*Removed [Oldschool-Milestone-Later](../labels/Oldschool-Milestone-Later) label.*

**Comment by DartBot** [comment]
<a href="https://github.com/munificent"><img src="https://avatars.githubusercontent.com/u/46275?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [munificent](https://github.com/munificent)**

----

*Marked this as being blocked by dart-lang/sdk#21082.*

**Comment by nex3** [comment]
`dart2dart` died long ago.
